test/test_core.py

```diff
@@ -2838,6 +2838,14 @@ def test_pthread_proxying_dropped_work(self):
     self.set_setting('PTHREAD_POOL_SIZE=2')
     self.do_run_in_out_file_test('pthread/test_pthread_proxying_dropped_work.c')
 
+  @node_pthreads
+  def test_pthread_proxying_canceled_work(self):
+    self.set_setting('EXIT_RUNTIME')
```

Comment:
Yes, otherwise the test hangs because of the bug with `runtimeKeepaliveCounter` and thread exit we investigated the other day where that change from a year ago needed to be partially reverted. (I don't think we ever filed an issue about that, unfortunately. Do you remember what I'm talking about?)

Reply:
Can you add a comment there (and/or a bug)?

Reply:
Filed #18782 and will add a comment to the test.

Reply:
This should no longer be needed I think how that that issue is fixed?

Reply:
Removed along with another instance of EXIT_RUNTIME in #18873.